README.md

```diff
@@ -136,11 +136,18 @@ provider  "ccx" {
 	client_id  =  "your_ccx_client_id"
 	client_secret  =  "your_ccx_client_secret
 	# base_url = "optionally_use_a_different_base_url"
+	# timeout = "15m"
 }
 ```
 
 > **Note:**
+> 
 > the option `base_url` may be used to specify a different ccx compliant cloud service.
+> 
+> The option `timeout` may be used to specify a different timeout for operations.
+> Default is `"15m"`.
```

Comment:
might be too short for big deployments (5 nodes and such)?

Reply:
increased to 60m